tests/TestResults/index.test.ts

```diff
@@ -1,9 +1,10 @@
 jest.disableAutomock()
+import * as tr from '../../src/TestResults'
 
 describe('TestResults', () => {
   describe('module.exports', () => {
     it('should match the snapshot', () => {
-      expect(require('../../src/TestResults')).toMatchSnapshot()
+      expect(tr).toMatchSnapshot()
```

Comment:
We can probably delete this file and the snapshots too. As much as it may help keep the internal API stable, this isn't testing the behaviour of the system. Sorry for my past mistakes. 🤮

Reply:
We can also do this in another PR.

Reply:
removed

Reply:
Thanks!